### ISSUE TITLE
[aws-vpc-cni] Do not use podLabels in the Label Selector

### ISSUE THREAD

**Issue by sidewinder12s** [aws/eks-charts#639]
**Describe the bug**
If a user needs to modify the labels used by pods, you cannot change them on a running deployment without a forced deployment, which is disruptive/not a rolling update.

The podLabels probably shouldn't be passed into the labelSelector field, which is not mutable. 

https://github.com/aws/eks-charts/blob/f4be91b5ae4a2959e821940a77d50dd0424841c1/stable/aws-vpc-cni/templates/daemonset.yaml#L17

**Steps to reproduce**

Add/Change the podLabels value.

**Expected outcome**
 
I'd expect to be able to change pod labels without a forced redeployment of the daemonset.

**Environment**

* Chart name: aws-vpc-cni
* Chart version:  1.1.10
* Kubernetes version: 1.19
* Using EKS (yes/no), if so version?
Yes, 1.19

**Additional Context**:

Not sure if the intention is to just use `originalMatchLabels` to avoid this, though I believe that option was a recent addition.

**Comment by jdn5126** [comment]
@sidewinder12s yeah, `originalMatchLabels` is the only current mechanism to avoid this
